src/main/java/com/ibr/fedora/testsuite/ExternalBinaryContent.java

```diff
@@ -20,37 +20,55 @@
  */
 package com.ibr.fedora.testsuite;
 
+import java.io.FileNotFoundException;
+import java.io.PrintStream;
+
+import org.testng.SkipException;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
 import com.ibr.fedora.TestSuiteGlobals;
 import com.ibr.fedora.TestsLabels;
+
 import io.restassured.RestAssured;
 import io.restassured.config.LogConfig;
 import io.restassured.http.Header;
 import io.restassured.http.Headers;
-import org.testng.SkipException;
-import org.testng.annotations.Parameters;
-import org.testng.annotations.Test;
-
-import java.io.FileNotFoundException;
-import java.io.PrintStream;
 
 public class ExternalBinaryContent {
 public TestsLabels tl = new TestsLabels();
-
+public String username;
+public String password;
     public String binary = "https://www.w3.org/StyleSheets/TR/2016/logos/UD-watermark";
     public String binary2 = "https://wiki.duraspace.org/download/attachments/4980737/"
 + "atl.site.logo?version=3&modificationDate=1383695533307&api=v2";
 
+/**
+ * Authentication
+ * @param username
+ * @param password
+ */
+@BeforeClass
+@Parameters({"param2", "param3"})
+public void auth(final String username, final String password) {
+this.username = username;
+this.password = password;
+}
+
     /**
+     * 3.8-A
      * @param host
      */
-    @Test(priority = 19)
+    @Test(priority = 34)
```

Comment:
Ideally, we would not need to enforce an order to the tests. Each test should create its own scaffolding.
...something to keep in mind for a future refactoring.

Reply:
We are looking for a solution that will use the order in which classes are added to the test to TestNG as the test order. If the priority is not included the test results are order and grouped by status, FAIL first, PASS second. We will continue to look for the solution.

Reply:
Thanks. It seems like a maintenance burden to have to be updating lots of `priority` values when new tests are added... but I also see the benefit in being able to compare two test results side-by-side, which would be difficult with different orderings.

It would be nice if this could be handled on the reporting side, and not in the tests... but this is not a critical issue.

Reply:
But I agree with Andrew:  not a high priority right now.